Makefile

```
@@ -312,6 +322,6 @@ pao-clean-e2e:
 
 
 .PHONY: cluster-node-tuning-operator-test-ext
-cluster-node-tuning-operator-test-ext:
+cluster-node-tuning-operator-test-ext: $(BINDATA_OTE)
```

checkmake (0.3.2)

[warning] Required target "test" is missing from the Makefile.

(minphony)